pheget/frontend/templates/frontend/variant.html

```diff
@@ -19,41 +18,170 @@
+              <input type="radio" name="label-options" autocomplete=off value="5"> Top 5
+            </label>
+            <label class="btn btn-secondary" data-toggle="tooltip" data-placement="top" data-html="true"
+                   title="If viewing P-values, add labels to the 50 eQTLs with the most significant P-values <b>if they are most significant than 10<sup>-20</sup></b>. If viewing Effect Sizes, choose the eQTLs with the 50 largest absolute effect sizes and only label those with P-value more significant than 10<sup>-20</sup>.">
+              <input type="radio" name="label-options" autocomplete=off value="50"> Top 50
+            </label>
+          </div>
+        </form>
+      </div>
+    </div>
+
+    <!-- Filters all points by TSS. Applies a region filter which is symmetric around the current point -->
+    <div class="row justify-content-start">
+      <div class="col-12" style="margin-bottom:0.8em">
+        <form class="tss-range" id="tss-both-range">
+          <div class="btn-group btn-group-toggle" data-toggle="buttons">
+            <span class="d-inline-block" tabindex="0" data-toggle="tooltip" data-html="true"
+                  title="Display eQTLs for genes <b>only</b> if their Transcription Start Sites (TSS's) are within the selected distance from this variant.">
+              <button type="button" class="btn btn-outline-secondary" style="pointer-events: none;">
+                <span class="fa fa-arrows-alt-h"></span> Filter by TSS Distance
+              </button>
+            </span>
+            <label class="btn btn-secondary" data-toggle="tooltip" data-placement="top"
+                   title="Show eQTLs for genes with TSS in the range chr{{ chrom }}:{{ '{:,}'.format([pos - 20000, 1] | max) }}-{{ '{:,}'.format(pos + 20000) }}">
+              <input type="radio" name="tss-options" autocomplete=off value="20000"> ±20kb
+            </label>
+            <label class="btn btn-secondary" data-toggle="tooltip" data-placement="top"
+                   title="Show eQTLs for genes with TSS in the range chr{{ chrom }}:{{ '{:,}'.format([pos - 50000, 1] | max) }}-{{ '{:,}'.format(pos + 50000) }}">
+              <input type="radio" name="tss-options" autocomplete=off value="50000"> ±50kb
+            </label>
+            <label class="btn btn-secondary" data-toggle="tooltip" data-placement="top"
+                   title="Show eQTLs for genes with TSS in the range chr{{ chrom }}:{{ '{:,}'.format([pos - 100000, 1] | max) }}-{{ '{:,}'.format(pos + 100000) }}">
+              <input type="radio" name="tss-options" autocomplete=off value="100000"> ±100kb
+            </label>
+            <label class="btn btn-secondary" data-toggle="tooltip" data-placement="top"
+                   title="Show eQTLs for genes with TSS in the range chr{{ chrom }}:{{ '{:,}'.format([pos - 200000, 1] | max) }}-{{ '{:,}'.format(pos + 200000) }}">
```

Comment:
Side thought- if we are repeating the same display transformation a lot (like range), we could add some sort of custom jinja tag/filter. This sort of trick (deduplicating code)  would mostly be useful if we wanted to keep UI consistent, but it's not in any way urgent.